applications/GeoMechanicsApplication/custom_utilities/solving_strategy_factory.hpp

```diff
@@ -21,91 +21,101 @@
 #include "scheme_factory.hpp"
 #include "convergence_criteria_factory.hpp"
 #include "builder_and_solver_factory.hpp"
+#include "solving_strategies/strategies/line_search_strategy.h"
 
-namespace Kratos
-{
+namespace Kratos {
 
 template<class TSparseSpace, class TDenseSpace, class TLinearSolver>
-class SolvingStrategyFactory
-{
+class SolvingStrategyFactory {
 public:
-    [[nodiscard]] static std::unique_ptr<SolvingStrategy<TSparseSpace, TDenseSpace>> Create(const Parameters& rSolverSettings, ModelPart& rModelPart)
-    {
+    [[nodiscard]] static std::unique_ptr<SolvingStrategy<TSparseSpace, TDenseSpace>>
+    Create(const Parameters &rSolverSettings, ModelPart &rModelPart) {
```

Comment:
We generally have the '&' attached to the type 
```suggestion
    Create(const Parameters& rSolverSettings, ModelPart& rModelPart) {
```

Reply:
Indeed, done